definitions/contested/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -3849,7 +3849,7 @@
     "TabLabel" : "Scheduling and Listing",
     "TabDisplayOrder" : 15,
     "CaseFieldID" : "chooseCourtLabelSL",
-    "FieldShowCondition" : "hearing_regionList!=\" \"",
```

Comment:
These change will only be visible to caseworker only. There are 6 tabs based on roles. Please check i think It has to be visible to all eligible parties.

Reply:
sorted on latest commit